loops.js

```diff
@@ -17,8 +25,18 @@
 // loop...
 // klaar!
  // ==========================================
+ console.log(" ================= Opdracht 2 =========================")
+
+ const loop = "loop...";
 
+for (let i = 0; i < 4; i++) {
+  console.log(loop);
 
```

Comment:
Dit werkt maar het is netter om een if en else te gebruiken. Dus if i < 3 loop... else “klaar!"